src/assets/css/main.less

```diff
@@ -195,13 +202,16 @@ body { display: flex; flex-direction: column; }
 
 /* missing styling for forms */
 .ui.form { margin-bottom: 1rem; }
+#half { width: calc(48% - 1ex); margin-right: 1ex; }
 .ui.form:last-child { margin-bottom: 0; }
 .ui.inline.form { display: inline-block; }
```

Comment:
Tried many, many things and cannot get the styles to apply to elements in the form to limit their width (ran webpack, etc).